appveyor.yml

```diff
@@ -15,7 +15,15 @@ cache:
 
 # Adapt as necessary starting from here
 
+environment:
+  GITHUB_PAT:
+    secure: Nkvazgyo9FBXlhdY1vdqGsSNJl/DjKXhBYRFd105iEpZxKAXWI5X1aQ/awLgCfWa
+  NOT_CRAN: true
+  USE_RTOOLS: true
+
 build_script:
+  - travis-tool.sh install_github r-lib/devtools
```

Comment:
Why do you need to install devel devtools and testthat?

Reply:
Maybe I don't, I saw failures that looked similar to what I see for `devtools:test()` with CRAN devtools and testthat, but then I was probably wrong.  Let me remove it.

Reply:
We'll see in ~10 minutes: https://ci.appveyor.com/project/gaborcsardi/pkginstall/build/1.0.21

Reply:
Of course it is already in the cache.... anyway, we'll be fine without it now....